### PR TITLE
Add full timestamp as tooltip for comments

### LINKED ISSUE #2260: Add tooltips for comment timestamps with the full timestamp
The timestamps on comments are human-readable but somewhat vague:

Screenshot of a comment with a timestamp of "15 days ago"

The Dates section on the right side of the page is the same but, when you mouseover the timestamp, a tooltip pops up with the full timestamp:

Screenshot of the Dates section with a timestamp of "15 days ago" and a tooltip containing a full timestamp

It would be nice if this functionality was extended to the comment timestamps.

### PR DESCRIPTION
**Pull request by elyscape**
Resolves #2260.

Signed-off-by: Eli Young <elyscape@gmail.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
jenkies test

File: bodhi/server/templates/fragments.html [hunk 1]
```diff
@@ -7,7 +7,9 @@
           ${util.markup(comment.text) | n}
         </div>
         <div class="col-md-2 text-xs-right text-muted">
-          <a href="${comment.url()}">${util.age(comment.timestamp)}</a>
+          <a href="${comment.url()}" data-toggle="tooltip" title="${comment.timestamp.strftime("%Y-%m-%d %H:%M:%S")}">
```
Comment (bowlofeggs) [inline bodhi/server/templates/fragments.html]: Some of the timestamps in the right hand box specify that the timestamp is UTC in their hovertexts (like the "Submitted" and "In testing" times") - would you like to add that here too?

Weirdly, it seems like the number of days to stable one doesn't have a hoverover.

Anyways, I'm fine with this patch as-is if you like this, but those are just some thoughts. Let me know!

**Review by bowlofeggs — APPROVED** [review]

File: bodhi/server/templates/fragments.html (hunk 1 above)
Comment (elyscape) [reply bodhi/server/templates/fragments.html]: Done. Should I add a hovertext to the days-to-stable entry?

Comment (bowlofeggs) [reply bodhi/server/templates/fragments.html]: @elyscape it's up to you - if you want to go for it.

**Comment by bowlofeggs** [comment]
jenkies test

File: bodhi/server/templates/fragments.html (hunk 1 above)
Comment (elyscape) [reply bodhi/server/templates/fragments.html]: Actually, looks like this was done on purpose. `update.days_to_stable` is a property that returns the number of days left until the update is eligible for stable. To add a tooltip to it, I'd either need to use that to calculate the time until stable in the template or add another property to the `Update` class that returns that value. As such, I'll leave it out, and we can consider adding it in a separate pull request.

**Comment by bowlofeggs** [comment]
On 05/07/2018 02:34 PM, Eli Young wrote:
> As such, I'll leave it out, and we can consider adding it in a separate
> pull request.

+1 - I'd forgotten about that. Thanks for looking into it, and I think
it's fine to not worry about it.

**Comment by elyscape** [comment]
@bowlofeggs Just checking to make sure this didn't fall through the cracks.

**Review by bowlofeggs — APPROVED** [review]

**Comment by bowlofeggs** [comment]
Sorry for taking so long to reply - things have been on $FIRE lately and I've been overwhelmed.

**Comment by elyscape** [comment]
No worries. Thank you very much!

**Comment by bowlofeggs** [comment]
This patch is planned to be included in the upcoming 3.9.0 release: #2477.

**Comment by bowlofeggs** [comment]
The patch for this issue was deployed to Fedora's production Bodhi instance yesterday.

**Pull request merged**